packaging/tests/tas-restapi/src/main/java/org/alfresco/rest/core/assertion/ModelAssertion.java

```diff
@@ -333,6 +333,22 @@ public T contains(String value)
             return (T) model;
         }
 
+        public T containsOnce(String value)
+        {
+            final String fieldContent = fieldValue.toString();
+            final int i = fieldContent.indexOf(value);
+            if (i == -1)
+            {
+                Assert.fail(errorMessage("does NOT contain once expected value: " + value + ", Current Value: " + fieldValue.toString()));
```

Comment:
Is the errorMessage here supposed to read "does NOT contain the expected value more than once"?

Reply:
Fixed to `does NOT contain at all the expected value`